Sources/TSCBasic/DictionaryExtensions.swift

```diff
@@ -9,22 +9,6 @@
 */
 
 extension Dictionary {
-    /// Convenience initializer to create dictionary from tuples.
-    public init<S: Sequence>(items: S) where S.Iterator.Element == (Key, Value) {
```

Comment:
The semantics don't really match with `uniqueKeysWithValues` since `uniqueKeysWithValues` will trap if there's a  duplicate key but this will just use the last key in the sequence. Are you confident that all these call sites will indeed have unique keys?

Reply:
I updated the PR to use `Dictionary(tuples, uniquingKeysWith: { $1 })` when necessary. I think we're okay now, but could you give the PR a quick check?